mne/_fiff/pick.py

```diff
@@ -1391,6 +1393,11 @@ def _picks_str_to_idx(
                 kwargs["fnirs"] = list(fnirs)[0]
             else:
                 kwargs["fnirs"] = list(fnirs)
+        if len(eyetrack) > 0 and not kwargs.get("eyetrack", False):
+            if len(eyetrack) == 1:
+                kwargs["eyetrack"] = list(eyetrack)[0]
+            else:
+                kwargs["eyetrack"] = list(eyetrack)
```

Comment:
I have a bias against if/else constructs where the code in each branch is very very similar, so this suggestion is maybe DRY enough to become obfuscated?  So treat as an optional suggestion:
```suggestion
            idx = 0 if len(eyetrack) == 1 else slice(None)
            kwargs["eyetrack"] = list(eyetrack)[idx]
```
... but if you do it, it should also be propogated to the `fnirs` case to make clear that they're parallel / doing the same thing

Reply:
It's a little harder for _me_ to comprehend bc I wasn't aware of the `slice(None)` indexing trick,  but no strong feelings from my side, I can commit the suggestion for both fnirs and eyetrack